miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -26,7 +27,8 @@ data class MiniAppSdkConfig(
     val isPreviewMode: Boolean,
     val requireSignatureVerification: Boolean = false,
     val miniAppAnalyticsConfigList: List<MiniAppAnalyticsConfig> = emptyList(),
-    val sslPinningPublicKeyList: List<String> = emptyList()
+    val sslPinningPublicKeyList: List<String> = emptyList(),
+    val requireAdPlacementBeta: Boolean = false
```

Comment:
Most of the MiniAppSdkConfig settings can also be set via AndroidManifest, so shouldn't this also be added in `MiniAppSdkInitializer`?

Also, it's fine to have the default be `false`, but I think we should set this to `true` in the Demo App settings so that it can be testable in QA.